index.html

```diff
@@ -154,7 +154,48 @@ <h3 class="skills-head"><img class="skill-bullets" src="./Work-section-img/squar
           </div>
         </div>
         </div>
+        </div>
+         
+        <div class="contact">
+            <div class="contact-grid">
+            <div class="contactpara">
+                <p class="contactp">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <div class="form">
+                <form class="form1" action="https://formspree.io/f/mgebknld" method="post">
+                    <div class="name">
+                    <input type="text" name="username" placeholder="Fullname" class="name1" maxlength="30" required>
+                    <div class="blank1"></div>
+                    </div>
+                    <div class="email">
+                    <input type="email" name="Email-address" placeholder="email address" class="email1" required>
+                    <div class="blank2"></div>
+                    </div>
+                    <div class="text">
+                    <textarea name="comment" placeholder="Enter text here" class="text1" maxlength="500" required></textarea>
+                    </div>
+                    <button name="submit" class="submit">Get in touch</button>
```

Comment:
- [ ] Nice work with the hover properties 🙂, However somethings are still left in the hover design 😀

- Your design 
<img width="133" alt="wrong1" src="https://user-images.githubusercontent.com/41163571/220545594-3311f218-c156-4aab-adff-c0abc2a8449e.png">

- Figma design
<img width="79" alt="right1" src="https://user-images.githubusercontent.com/41163571/220545648-76576b23-d705-4691-bf56-e7a0e15edefa.png">

Did you spot the difference? 🌚.

Things like the border and the "Get in touch" text size and weight.